src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,122 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(nullable = false)
+    private String isbn;
+
+    @Column(nullable = false)
+    private String author;
+
+    @Column
```

Comment:
Curiosity (just so you know it's existence): you can avoid using the annotation @Column and it will be mapped to the database with the attribute's name.

Reply:
But I think that If I omit the annotation, on DB it will accept `null`s